test/classification/test_snorkel_classifier.py

```diff
@@ -45,12 +90,37 @@ def test_twotask_partial_overlap_model(self):
         self.assertEqual(len(model.module_pool), 3)
 
     def test_bad_tasks(self):
-        task1 = create_task("task1", module_suffixes=["A", "A"])
         with self.assertRaisesRegex(ValueError, "Found duplicate task"):
             SnorkelClassifier(tasks=[task1, task1])
         with self.assertRaisesRegex(ValueError, "Unrecognized task type"):
             SnorkelClassifier(tasks=[task1, {"fake_task": 42}])
 
+    def test_predict(self):
+        model = SnorkelClassifier([task1])
+        results = model.predict(dataloader)
+        self.assertEqual(sorted(list(results.keys())), ["golds", "probs"])
+        np.testing.assert_array_equal(
+            results["golds"]["task1"], dataloader.dataset.Y_dict["task1"].numpy()
+        )
+        self.assertEqual(results["probs"]["task1"].shape, (NUM_EXAMPLES, 2))
+
+        results = model.predict(dataloader, return_preds=True)
+        self.assertEqual(sorted(list(results.keys())), ["golds", "preds", "probs"])
+        self.assertEqual(results["preds"]["task1"].shape, (NUM_EXAMPLES,))
+
+    def test_empty_batch(self):
+        # Make the first BATCH_SIZE labels 0 so that one batch will have no labels
+        dataloader = create_dataloader("task1", shuffle=False)
+        for i in range(BATCH_SIZE):
+            dataloader.dataset.Y_dict["task1"][i] = 0
+        model = SnorkelClassifier([task1])
+        model.predict(dataloader)
+
+    def test_score(self):
+        model = SnorkelClassifier([task1])
+        metrics = model.score([dataloader])
+        self.assertIsInstance(metrics["task1/dataset/train/accuracy"], float)
```

Comment:
What should the accuracy be?

Reply:
Unknown--it's a randomly initialized network.